projects/novo-elements/src/utils/form-utils/FormUtils.ts

```diff
@@ -587,6 +588,8 @@ export class FormUtils {
       return [{ value: false, label: this.labels.no }, { value: true, label: this.labels.yes }];
     } else if (field.workflowOptions && fieldData) {
       return this.getWorkflowOptions(field.workflowOptions, fieldData);
+    } else if (field.dataSpecialization === 'SPECIALIZED_OPTIONS') {
+      return field.options;
```

Comment:
we should be filtering out options with isEditOption = false

Reply:
is isEditOption going to be implemented in the future? Here is a current example {value: 3, label: "Weeks", readOnly: false}